usersession/userd/launcher.go

```diff
@@ -196,9 +199,23 @@ func (s *Launcher) OpenURL(addr string, sender dbus.Sender) *dbus.Error {
 		// scheme is not listed in our allowed schemes list, perform
 		// fallback and check whether the local system has a handler for
 		// it
-		isAllowed, err = schemeHasHandler(u.Scheme)
+		handler, err := schemeHandler(u.Scheme)
 		if err != nil {
 			logger.Noticef("cannot obtain scheme handler for %q: %v", u.Scheme, err)
+		} else if handler != "" {
+			// we have a handler, let's ask the user
+			dialog, err := ui.New()
+			if err == nil {
+				answeredYes := dialog.YesNo(
+					i18n.G("Allow opening a URL?"),
```

Comment:
I wouldn't put a question in the title; unless this has been decided by some UX designer, I'd vote for a text like "Resource permission request" or any permutation of 3 random nouns. Or even https://www.atrixnet.com/bs-generator.html ;-)